webapp/src/pages/Map.tsx

```diff
@@ -239,6 +256,28 @@ function MapPage() {
 
   const [expandedLayers, setExpandedLayers] = useState<string[]>([])
 
+  // Moove AI layer local state variables
+  const [mooveAiPolygonSource, setMooveAiPolygonSource] = useState<GeoJSON.Feature<GeoJSON.Geometry>>({
+    type: 'Feature',
+    geometry: {
+      type: 'Polygon',
+      coordinates: [],
+    },
+    properties: {},
+  })
+
+  const [mooveAiPolygonPointSource, setMooveAiPolygonPointSource] = useState<
+    GeoJSON.FeatureCollection<GeoJSON.Geometry>
+  >({
+    type: 'FeatureCollection',
+    features: [],
+  })
+
+  const [mooveAiDataSource, setMooveAiDataSource] = useState<GeoJSON.FeatureCollection<GeoJSON.Geometry>>({
+    type: 'FeatureCollection',
+    features: [],
+  })
```

Comment:
suggestion (non-blocking) - Move this logic to the mapSlice, this file is massive...

Reply:
Addressed